pages/favicons-database.md

```diff
@@ -0,0 +1,20 @@
+---
+title: OWASP Favicon Database
+layout: col-sidebar
+author:
+contributors:
```

Comment:
You could check the history of the wiki page content, and also add yourself.

Reply:
Updated it, check once.